build/sslstream-scenarios.yml

```diff
@@ -0,0 +1,88 @@
+# SslStream scenarios
+
+parameters:
+  - name: arguments
+    type: string
+    default: ""
+  - name: connection
+    type: string
+    default: ""
+  - name: condition
+    type: string
+    default: "true"
+
+  - name: readWriteScenarios
+    type: object
+    default:
+      - displayName: "read-write"
+        arguments: --scenario read-write --client.options.collectCounters true
+
+  - name: handshakeScenarios
+    type: object
+    default:
+      - displayName: "read-write"
+        arguments: --scenario handshake --client.options.collectCounters true

```

Comment:
```suggestion
        arguments: --scenario handshake --client.options.collectCounters true --property scenario=handshake
```

Reply:
Yes, the scenario property should be set, but this can be nicer, like "Handshake". This will be used in the PBI dashboard as a list to filter charts.